File: src/gatling/scala/org/openapitools/client/model/Sort.scala

package org.openapitools.client.model


case class Sort (
    /* Name of the property by which the result is sorted */
    _property: Option[String],
    /* Sort direction (ASC or DESC) */
    _direction: Option[String]
)
object Sort {
    def toStringBody(var_property: Object, var_direction: Object) =
        s"""
        | {
        | "property":$var_property,"direction":$var_direction
        | }
        """.stripMargin
}

File: client/zapier/models/Sort.js
const utils = require('../utils/utils');

module.exports = {
    fields: (prefix = '', isInput = true, isArrayChild = false) => {
        const {keyPrefix, labelPrefix} = utils.buildKeyAndLabel(prefix, isInput, isArrayChild)
        return [
            {
                key: `${keyPrefix}property`,
                label: `Name of the property by which the result is sorted - [${labelPrefix}property]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}direction`,
                label: `Sort direction (ASC or DESC) - [${labelPrefix}direction]`,
                type: 'string',
                choices: [
                    'ASC',
                    'DESC',
                ],
            },
        ]
    },
    mapping: (bundle, prefix = '') => {
        const {keyPrefix} = utils.buildKeyAndLabel(prefix)
        return {
            'property': bundle.inputData?.[`${keyPrefix}property`],
            'direction': bundle.inputData?.[`${keyPrefix}direction`],
        }
    },
}
